core/contracts/financial_templates/Liquidatable.sol

```diff
@@ -8,71 +8,41 @@ import "../FixedPoint.sol";
+dispute false liquidations and compensate position sponsors who had their position
+incorrectly liquidated. Importantly, a prospective disputer must deposit a dispute
+bond that they can lose in the case of an unsuccessful dispute.
+*/
 contract Liquidatable is PricelessPositionManager {
     using FixedPoint for FixedPoint.Unsigned;
     using SafeMath for uint;
 
     enum Status { PreDispute, PendingDispute, DisputeSucceeded, DisputeFailed }
 
     struct LiquidationData {
-        /**
-         * Following variables set upon creation of liquidation:
-         */
-
-        // When Liquidation ends and becomes 'Expired'
-        uint expiry;
-        // Person who created this liquidation
-        address liquidator;
-        // Liquidated (and expired or not), Pending a Dispute, or Dispute has resolved
-        Status state;
-        /**
-         * Following variables determined by the position that is being liquidated:
-         */
-
-        // Synthetic Tokens required to be burned by liquidator to initiate dispute
-        FixedPoint.Unsigned tokensOutstanding;
-        // Collateral locked by contract and released upon expiry or post-dispute
-        FixedPoint.Unsigned lockedCollateral;
-        // Amount of collateral being liquidated, which could be different from
+        // Following variables set upon creation of liquidation:
+        uint expiry; // When Liquidation ends and becomes 'Expired'
+        address liquidator; // caller who created this liquidation
+        Status state; // Liquidated (and expired or not), Pending a Dispute, or Dispute has resolved
+        // Following variables determined by the position that is being liquidated:
+        FixedPoint.Unsigned tokensOutstanding; // Synthetic Tokens required to be burned by liquidator to initiate dispute
+        FixedPoint.Unsigned lockedCollateral; // Collateral locked by contract and released upon expiry or post-dispute
+        FixedPoint.Unsigned liquidatedCollateral; // Amount of collateral being liquidated, which could be different from
         // lockedCollateral if there were pending withdrawals at the time of liquidation
-        FixedPoint.Unsigned liquidatedCollateral;
-        /**
-         * Following variables set upon a dispute request
-         */
-
-        // Person who is disputing a liquidation
-        address disputer;
-        // Time when dispute is initiated, needed to get price from Oracle
-        uint disputeTime;
-        // Final price as determined by an Oracle following a dispute
-        FixedPoint.Unsigned settlementPrice;
-    }
 
-    /**
-    * Contract-wide variables, consistent across all liquidations for synthetic tokens
-    * of this template
-    */
+        // Following variables set upon a dispute request
+        address disputer; // Person who is disputing a liquidation
+        uint disputeTime; // Time when dispute is initiated, needed to get price from Oracle
```

Comment:
I know you didn't change this, but aren't Oracle requests supposed to be made at liquidationTime not disputeTime?

Reply:
+1

Reply:
This is a really good point actually. This is wrong here and in the `createLiquidation` function. I will postpone this for a later PR and have created an issue #895